### PR TITLE
 feat: DHIS2-9276 #4 - filters without correlated columns

### PR DESCRIPTION
**Pull request by JoakimSM**
[DHIS2-9276](https://jira.dhis2.org/browse/DHIS2-9276)

Adding filters without a correlating column (i.e. filters that will not be found in the column list). An example would be the enrollment status: setting the filter to `ACTIVE` will return tracked entity instances that have any enrollments with active status.

### CODE REVIEW

File: src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/Setup/TeiWorkingListsSetup.component.js [hunk 1]
```diff
@@ -1,30 +1,108 @@
...
+                    return acc;
+                }, {});
+        } else if (filter.startsWith('ge')) {
+            queryArgs.programIncidentStartDate = filter.replace('ge:', '');
+        } else {
+            queryArgs.programIncidentEndDate = filter.replace('le:', '');
+        }
+        return queryArgs;
+    },
+}, {
```
Comment (JoakimSM) [inline src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/Setup/TeiWorkingListsSetup.component.js:72]: Pretty much the same as the one above, but not putting any effort into generalizing at this point. Probably will do it we need another one with the same structure

File: src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/types/concept.types.js [hunk 2]
```diff
@@ -56,6 +56,14 @@ export type TeiColumnMetaForDataFetching = {
 
 export type TeiColumnsMetaForDataFetching = Map<string, TeiColumnMetaForDataFetching>;
 
+export type TeiFilterOnlyMetaForDataFetching = {
```
Comment (JoakimSM) [inline src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/types/concept.types.js:59]: need to keep this inexact.

File: src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/epics/teiViewEpics/lib/initTeiWorkingListsView.types.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+export type Input = {|
     programId: string,
     orgUnitId: string,
     storeId: string,
     columnsMetaForDataFetching: TeiColumnsMetaForDataFetching,
+    filtersOnlyMetaForDataFetching: TeiFiltersOnlyMetaForDataFetching,
     querySingleResource: QuerySingleResource,
     absoluteApiPath: string,
-};
+|};
```
Comment (paschalidi) [inline src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/epics/teiViewEpics/lib/initTeiWorkingListsView.types.js:13]: 👍 great

**Review by paschalidi — APPROVED** [review]
Great work @JoakimSM thanks fot the short and focused PR! 👍

**Pull request merged**